src/openfl/display/DisplayObject.hx

```diff
@@ -1017,93 +1088,17 @@ class DisplayObject extends EventDispatcher implements IBitmapDrawable #if openf
 		
 		__worldTransformInvalid = false;
```

Comment:
I think overall I would like to call `__updateTraverse` to `__traverseDirty` and `__worldTransformInvalid` to `__worldTransformDirty` to call everything dirty things dirty oO. In the end we would have different dirty states, which could also go in one `int` or something as flags:

```__traverseDirty
__updateDirty
__transformDirty
__worldTransformDirty
__renderDirty```